Formula/bali-phy.rb

```diff
@@ -1,9 +1,11 @@
 class BaliPhy < Formula
   # cite Suchard_2006: "http://dx.doi.org/10.1093/bioinformatics/btl175"
+  # cite Redelings_2014: https://dx.doi.org/10.1093%2Fmolbev%2Fmsu174
```

Comment:
Could you please change both `%2F` to `/` ?

Reply:
Done.